crates/ironrdp-server/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "ironrdp-server"
+version = "0.1.0"
+readme = "README.md"
+description = ""
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+authors.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[lib]
+doctest = false
+test = false
+
+[dependencies]
+anyhow = "1.0"
+bytes = "1"
+tokio = { version = "1", features = ["full"] }
```

Comment:
Since `ironrdp-server` is used as a library, it’s better to not enable all tokio features and only enable the ones required by the crate.